code_quality_tools/detekt.yml

```diff
@@ -114,7 +114,7 @@ complexity:
     threshold: 600
   LongMethod:
     active: true
-    threshold: 60
+    threshold: 70
```

Comment:
Please roll it back.
If a function is so big it's probably a problem in implementation.